misk-aws/src/test/kotlin/misk/jobqueue/sqs/QueueResolverTest.kt

```diff
@@ -1,51 +0,0 @@
-package misk.jobqueue.sqs
-
-import com.amazonaws.services.sqs.AmazonSQS
-import com.amazonaws.services.sqs.model.GetQueueUrlRequest
-import com.amazonaws.services.sqs.model.GetQueueUrlResult
-import misk.cloud.aws.AwsAccountId
-import misk.cloud.aws.AwsRegion
-import misk.jobqueue.QueueName
-import misk.mockito.Mockito.mock
-import org.assertj.core.api.Assertions.assertThat
-import org.junit.jupiter.api.Test
-import org.mockito.ArgumentMatchers.any
-import org.mockito.Mockito.`when`
-
-internal class QueueResolverTest {
-
-  @Test
-  fun `getDeadLetter returns global DLQ when one is configured`() {
```

Comment:
One the fence about keeping one of these, not so much about testing multiple outputs but rather that it uses the specified DLQ provider. Thoughts?